pytest_fixtures/broker.py

```diff
@@ -1,12 +1,41 @@
 import pytest
 from broker.broker import VMBroker
+from wait_for import wait_for
 
 from robottelo.constants import BROKER_RHEL77
 from robottelo.hosts import Capsule
 from robottelo.hosts import ContentHost
 from robottelo.hosts import Satellite
 
 
+@pytest.fixture(scope='session')
+def satellite_factory():
+    def factory(retry_limit=3, delay=300, **broker_args):
+        vmb = VMBroker(host_classes={'host': Satellite}, workflow='deploy-sat-lite', **broker_args)
```

Comment:
The sat-lite template is going away, so this should be deploy-sat-jenkins.

That aside, it shouldn't be hard-coded here, this should be read from settings.